test/jdk/java/util/Locale/LocaleTest.java

```diff
@@ -677,7 +678,7 @@ public void TestUninstalledISO3Names() {
     }
 
     /**
-     * @bug 4052404 4778440
+     * @bug 4052404 4778440 8263202
      */
     public void TestChangedISO639Codes() {
```

Comment:
Could probably be simplified with a DataProvider.

Reply:
That would be nice, but the test is not testng based, and it would be an entire test rewrite which I would not do it at thist time.

Reply:
I see. Good old test is still good.